### ISSUE TITLE
Image is broken

### ISSUE THREAD

**Issue by sicutdeux** [inmagik/registryo#3]
Hey, thanks for all your work, currently I use Registryo on my k3s cluster, updated today as long overdue task, but, got this error message as the pod deployed...

```
160 static files copied to '/var/www/statik'.
2023-04-12T18:10:34.306011472-05:00 System check identified some issues:
2023-04-12T18:10:34.306106382-05:00 
2023-04-12T18:10:34.306126138-05:00 WARNINGS:
2023-04-12T18:10:34.306161870-05:00 main.ACLEntry: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
2023-04-12T18:10:34.306183786-05:00 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the MainConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
2023-04-12T18:10:34.636964779-05:00 Operations to perform:
2023-04-12T18:10:34.637037879-05:00   Apply all migrations: admin, auth, authtoken, contenttypes, main, sessions
2023-04-12T18:10:34.637056808-05:00 Running migrations:
2023-04-12T18:10:34.637075383-05:00   No migrations to apply.
2023-04-12T18:10:35.255410344-05:00 2023/04/12 23:10:35 [emerg] 11#11: "upstream" directive is not allowed here in /etc/nginx/conf.d/default.conf:2
2023-04-12T18:10:35.255478289-05:00 nginx: [emerg] "upstream" directive is not allowed here in /etc/nginx/conf.d/default.conf:2
```

So it seems the NGINX config is bad, the upstream seams outside the http holders.

Regards.

**Comment by osioalberto** [comment]
@sicutdeux thanks for your report, the image has been fixed and released as tag `v1.1.1` (the `latest` tag has been updated as well)

**Comment by sicutdeux** [comment]
Works perfectly again, thanks for your work and effort.
